nixos/modules/services/networking/strongswan.nix

```diff
@@ -81,6 +82,8 @@ in
   options.services.strongswan = {
     enable = mkEnableOption "strongSwan";
 
+    package = mkPackageOption pkgs "strongswan" { };
```

Comment:
Where is the package option used?

Reply:
It's not being used. The unit is referencing pkgs directly, and there is a with cfg there.

Reply:
> It's not being used. The unit is referencing pkgs directly, and there is a with cfg there.

Then these `pkgs.strongswan` should be replaced with `cfg.package`.

Reply:
Indeed, thanks :) I just failed at `git add -p`

Reply:
it's fixed now: 25ecc0ed58ca46f6d28afeacad0f7a20aacf3af0